index.bs

```diff
@@ -439,6 +440,157 @@ spec: webidl
     it MUST remove it.
     </li>
   </ol>
+
+  <section>
+    <h3 id="storage-manipulation">Storage Manipulation Algorithms</h3>
+
+    <p>
+      Other specifications can use the algorithms defined in this section to
+      manipulate the <a>permission store</a> from their algorithms.
+    </p>
+
+    <div class="example" id="example-storage-manipulation">
+      <p>
+        The [[notifications]] API can check for the user having granted
+        permission using:
+      </p>
+      <blockquote>
+        If the {{"notifications"}} <a>permission's state</a> is not
+        {{"granted"}}, reject <var>promise</var> with a {{TypeError}} exception
+        and terminate these substeps.
```

Comment:
It doesn't have to be, given `document.domain`. But maybe you always ignore the domain field...